workspaces/rbac/plugins/rbac-backend/src/policies/permission-policy.ts

```diff
@@ -91,6 +97,51 @@ export class RBACPermissionPolicy implements PermissionPolicy {
       'permission.rbac.conditionalPoliciesFile',
     );
 
+    // Read default user access configuration - properly handle Boolean values
+    const defaultUserAccessEnabled = configApi.getOptionalBoolean(
+      'permission.rbac.defaultUserAccess.enabled',
+    );
+    // Explicitly log the actual config value to help diagnose issues
+    logger.info(
+      `Default user access config value: ${defaultUserAccessEnabled}`,
+    );
+    logger.info(
+      `Default user access enabled: ${Boolean(defaultUserAccessEnabled)}`,
+    );
```

Comment:
Seems like this is roughly logging the same thing twice. Is there anyway we can simplify this down to one log?